comictalker/resulttypes.py

```diff
@@ -26,3 +26,9 @@ class ComicIssue(TypedDict, total=False):
     site_detail_url: str
     volume: ComicVolume
     alt_images_url: str  # Comma separated URLs
+    characters: str  # Newline separated
+    locations: str  # Newline separated
+    credits: str  # JSON: "{"name": "Bob Shakespeare", "role": "Writer"}"
+    teams: str  # Newline separated
+    story_arcs: str  # Newline separated
+    complete: bool  # Is the data complete? Includes characters, locations, credits.
```

Comment:
All of these should be actual lists and the json should be a dictionary or class

Reply:
I did the conversion in comiccacher as that seemed to make the most sense. I was sure if doing:
```
                    "characters": "\n".join(issue["characters"]),
                    "locations": "\n".join(issue["locations"]),
                    "teams": "\n".join(issue["teams"]),
                    "story_arcs": "\n".join(issue["story_arcs"]),
                    "credits": json.dumps(issue["credits"]),
```
was the "correct" way to do it or if it's better to make a var and do it that way?

Reply:
That is perfect!